shared/githubDataStore.ts

```diff
@@ -1,25 +1,39 @@
 import { Octokit } from "@octokit/core";
+import { OctokitResponse } from "@octokit/types";
+
+import { createCache } from "./cache";
 
 const githubOwner = "theramis";
 const githubRepo = "aucvid";
 
 const octokit = new Octokit({ auth: process.env.GITHUB_PAT });
 
+type LatestFetchyRunResponse = OctokitResponse<
+  { workflow_runs: { run_started_at?: string | undefined }[] },
+  200
+>;
```

Comment:
Had to create this type to populate the cache - is this what I should / shouldn't be doing?